File: api/src/main/java/me/devsnox/planetsystem/api/player/OfflinePlanetPlayer.java
package me.devsnox.planetsystem.api.player;

import me.devsnox.planetsystem.api.planet.Planet;

import java.util.List;
import java.util.UUID;

public interface OfflinePlanetPlayer {

    UUID getUUID();

    Planet getPlanet();

    List<Planet> getMemberedPlanets();
}

File: api/src/main/kotlin/de/astride/planetsystem/api/player/OfflinePlanetPlayer.kt
package de.astride.planetsystem.api.player

import de.astride.planetsystem.api.inline.Owner
import de.astride.planetsystem.api.planet.Planet


//TODO Check (planet has already the owner)
interface OfflinePlanetPlayer {

    val owner: Owner

    val planet: Planet

    fun load(request: (PlanetPlayer) -> Unit)

}
